pkgs/tools/networking/nexttrace/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "nexttrace";
+  version = "1.1.2";
+  src = fetchFromGitHub {
+    owner = "sjlleo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-hlu5uflbiftD1MKcECsRUZPrJIq5q+WQ70y1QoG5Guc=";
+  };
+  vendorSha256 = "sha256-SA22qjPS9c5MDKBA0YpL6BmFp/7/RGX0NYjNiKce6rM=";
+  doCheck = false; # its tests require networking
+
+  meta = with lib; {
+    description = "An open source visual route tracking CLI tool";
+    homepage = "https://mtr.moe";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ sharzy ];
+    platforms = platforms.unix;
```

```suggestion
    platforms = platforms.linux;
```
If this requires a module shouldn't this be set to `linux`?

---

For non-nixos user, the user can call nexttrace with `sudo` or something. Similar tools, e.g. `mtr`, `inettools`, also take `platforms.unix`. Hence I think `unix` is ok.